src/common/sidebar/components/SearchResults.tsx

```diff
@@ -22,7 +22,7 @@ const SearchResults = ({ hits }: Props) => {
 
   const handleClick = (e: MouseEvent<HTMLDivElement>) => {
     const pid = e.currentTarget.getAttribute('data-pid');
-    if (pid) {
+    if (pid && scheduleMatch === null) {
```

Comment:
```suggestion
    if (pid && !scheduleMatch) {
```
Just a small simplification